src/routes/v2/authenticatedSites/__tests__/Settings.spec.js

```diff
@@ -92,7 +92,10 @@ describe("Settings Router", () => {
       homepage,
     })
 
-    const updatedFbPixelValue = `${configContent["facebook-pixel"]}test`
+    const updatedFbPixelValue = parseInt(
+      `${configContent["facebook-pixel"]}0`,
```

Comment:
basic qn, why got "0" at the back ah?

Reply:
This is to represent a changed value! it used to have `test` appended to it, but since it's now of type number the value needs to still be a number